shared/chat/conversation/input-area/normal/mention-input.js

```diff
@@ -98,18 +104,25 @@ class MentionInput extends React.Component<MentionInputProps, MentionState> {
         this._setChannelMentionFilter(word.substring(1))
       }
     } else if (selection.start !== selection.end) {
-      this.state.mentionPopupOpen && this._setMentionPopupOpen(false) && this._setMentionFilter('')
+      this.state.mentionPopupOpen &&
```

Comment:
hunh, I think the old line here is either buggy or works only by coincidence. `_setMentionPopupOpen` doesn't return anything meaningful, so I think what is intended is:
```
if (this.state.mentionPopupOpen) {
  this._setMentionPopupOpen(false)
  this._setMentionFilter('')
}
```
probably don't need to clear the last selection key, either.

Can you fix the instance below and also do a quick check that there aren't any other instances of this?